cloud-db/field-acl.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Field-based ACL
+---
+
+[[toc]]
+
+You can specify ACL to a record's field.
+Similar to [Record-based ACL][doc-record-acl], you can set access level and
+user target to Field-based ACL.
+
+Field-based ACL is a more advanced feature of Skygear, it gives finer control
+to the access, with [Discovery Level][doc-field-acl-discovery-level] and
+[More User Target][doc-field-acl-user-target].
+
+Field-based ACL also provide control on [Access Level][doc-access-level].
+
+## Setting Field-based ACL
+
+The simplest way to edit Field-based ACL is to use Portal.
+
+![Field ACL & Schema](/assets/common/field-acl-and-schema.png)
+
+## ACL Discovery Level
+
+- `Queryable`: Fields that can be used to perform any query predicates.
+
+- `Discoverable`: Fields that can only be used to perform equality query predicate.
+Supported operators: `equalTo`, `contains`(or `=` and `IN` for predicate in iOS).
+
+- `Not Queryable`: Fields that not available to perform query.
+
+## Advanced ACL User Target of Field-based ACL
+
+- `Owner`: The owner of the record instance.
+
+- `Specific User`: The user specified by user ID.
+
+- `Fields with User Objects`: The users specified in a field of the records.
+
+- `Role`: The user-defined roles.
+
+- `Any Users`: Any logged-in users.
+
+- `Public`: Any users with correct API key, including not authenticated users.
+
+## Record Partial Update
+
+There are 2 cases when calling record save API:
+
+### atomic: `True`
+
+If you change a field that you do not have access to write, those field will
+not be changed, while others will be updated.
+
+The SDK will provide partial update callback for record save API.
+
+### atomic: `False`
+
+You will simply get an error, for not having access right to update the field.
```

Comment:
The `atomic` section need to link back to `db.save` related description. And I think an example is needed.